README.md

```diff
@@ -288,27 +434,70 @@ import { getLocalePack } from '@americanexpress/one-app-ducks';
 // ...
 
 store.dispatch(getLocalePack('en-GB'));
+```
+
+### `redirection` duck
```

Comment:
does this still work after removing react-router-redux ?

Reply:
Yes, this did not rely on react-router-redux